playbooks/support/responding.md

```diff
@@ -26,6 +26,9 @@ respond to these issues.
    - Take your best guess at the other information, it can always be edited later.
 
 2. Using the slack integration on the Jira Ops ticket, create a new slack channel to discuss resolution.
+
+   _Jira integrations are known to be flakey. If you have trouble getting the integration to work, simply create a channel named `#incident-N` (where `N` corresponds to the Jira Ops ticket) and link to it from the ticket._
```

Comment:
Oh gosh... this didn't work either? 😭

Reply:
It didn’t for @izakp, it did for me but that seems like the definition of flakey :)

Reply:
It worked when I created Incident-31 but yes, the integration is definitely flakey